packages/components/src/styles/footnote/footnote.stories.ts

```diff
@@ -0,0 +1,147 @@
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <h4 class="sd-headline sd-headline--size-base">New Context</h4>
+
+    <p class="sd-footnote sd-footnote--number sd-footnote--reset">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+    </div>
+    </div>`;
+  }
+};
+
+/**
+ * This sample shows how to use the `--number` and `--reset` classes in combination. The `--number` class should be included in every footnote that needs a number. The `--reset` class should be included at the first `sd-footnote` to indicate that it's the first element. It can be used again later on to restart the counter from 1.
+ */
+
+export const Sample = {
+  parameters: { controls: { include: [] } },
+  name: 'Sample: Counter',
+  render: () => {
+    return html`
+      <div class="sd-container flex flex-col gap-4">
+        <p class="sd-footnote sd-footnote--number sd-footnote--reset">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliq erat, sed diam voluptua.
+        </p>
+
+        <p class="sd-footnote sd-footnote--number">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliq erat, sed diam voluptua.
+        </p>
+
+        <p class="sd-footnote sd-footnote--number">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliq erat, sed diam voluptua.
+        </p>
+
+        <p class="sd-footnote sd-footnote--number sd-footnote--reset">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliq erat, sed diam voluptua.
+        </p>
+
+        <p class="sd-footnote sd-footnote--number">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliq erat, sed diam voluptua.
+        </p>
+      </div>
+    `;
+  }
+};
```

Comment:
Here the same: https://github.com/solid-design-system/solid/pull/863/files#r1524569936